src/hextools/utils.py

```diff
@@ -11,3 +15,24 @@ def now():
 def replace_curlies(string, how_many=2):
     """A helper function to replace multiple curly braces with one."""
     return string.replace("{" * how_many, "{").replace("}" * how_many, "}")
+
+
+def get_ioc_hostname(pvname):
```

Comment:
Did we want to think about moving this function to a more widely used library, i.e. `nslsii`

Reply:
Good idea! Let's do it outside of this PR (maybe we can have a copy here to minimize the number of dependencies as `nslsii` becomes a bit too heavy).

Reply:
Done: https://github.com/NSLS-II/nslsii/pull/182.